storage/backup/backup-cli/src/coordinators/backup.rs

```diff
@@ -345,6 +351,7 @@ impl BackupCompactor {
         metadata_cache_opt: MetadataCacheOpt,
         storage: Arc<dyn BackupStorage>,
         concurrent_downloads: usize,
+        remove_compacted_files_after: u64,
```

Comment:
nit: rename to remove_compacted_files_after_secs, or use duration/timestamp struct here and other places, or at least add a comment.

Reply:
Done